microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+* [ICoreWebView2Profile4 interface](#)
+	* [ICoreWebView2Profile4::SetPermissionState method](#)
+	* [ICoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [ICoreWebView2PermissionSettingCollectionView interface](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [ICoreWebView2PermissionSetting interface](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [ICoreWebView2ControllerOptions2 interface](#)
+	* [ICoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [ICoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [ICoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [ICoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+---
+
+#### Bug fixes
+ 
+*   Fixed a bug where WebView2 was not closing properly when hitting a BeforeUnload event (Runtime) ([Issue #2677](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2677)) 
+*   In the DownloadStarting event, the ResultFilePath previously was not showing the correct download location for UWP applications when the DownloadStarting event handler was attached.This has been fixed and the correct ResultFilePath is shown.  
+*   Fixed bug of throwing System.ArgumentException after call HostObject method that returns non-generic task.([Issue #2787](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2787))   
+*   Fixed an issue in SharedBuffer API where the stream object does not work well with StreamWriter. (Runtime) ([Issue #3108](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3108)) 
+*   DOM speech synthesis APIs like `SpeechSynthesis.getVoices()` will now work in UWP apps. (Runtime)  
+*   Fixed a crash on frame destruction. (Runtime)([Issue #3062](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3062))   
+*   Fixed a bug where the app crashes when trying to call CreateWebResourceResponse with null phrase reason. This is a runtime change.  
+*   The `CoreWebView2.AddHostObjectToScript` option `chrome.webview.hostObjects.options.ignoreMemberNotFoundError` now works in non-English locales. (Runtime)  
+*   Fully enabled Open file dialog support for elevated apps on Windows 7. 
+*   Fixed a bug where owned windows were not showing up for UWP.
```

Comment:
```suggestion
*   Fixed a bug where owned windows were not appearing for UWP.
```
alternatives - use the most accurate, specific wording:
*   Fixed a bug where owned windows were not opening for UWP.
*   Fixed a bug where owned windows were not being rendered for UWP.
*   Fixed a bug where owned windows were not being created for UWP.